.ci/scripts/package-docker-snapshot.sh

```diff
@@ -11,7 +11,8 @@ set -euo pipefail
 NEW_TAG=${1:?Docker tag is not set}
 NEW_IMAGE=${2:?Docker image is not set}
 
-export PLATFORMS='linux/amd64'
+# linux/amd64 is in the default list already
+export PLATFORMS="${PLATFORMS:-+linux/amd64}"
```

Comment:
IIRC `PLATFORMS="+linux/amd64"` will build the defaults platform plus `linux/amd64`, if you set `PLATFORMS='linux/amd64'` it build only `linux/amd64`

Reply:
I dunno which behaviour we want here

Reply:
In fact, I don't need this anymore, it was wrongly copied from another PR that I worked with